.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+* @davemacaulay @kshorten42 @JosephLeedy @NickolasMalovanets @pavel-bystritskyi
```

Comment:
Hello @JosephLeedy,
pavel-bystritskyi is no longer in our team and should be excluded from this repo.

Reply:
Done. Thank you for informing me; I was unaware of his departure.